app/init.php

```diff
@@ -196,6 +198,22 @@ function($value) {
     }
 );
 
+Database::addFilter('enum',
```

Comment:
Not sure why we need a filter? We can just store the element options inside formatOptions, and leave validation to our standard validation flow.

Reply:
We need a way to extract values from `formatOptions` and add them into the response model. This `enum` filter ensures:
- we don't clash with the Structure validator when writing the attribute document to the internal table
- Extracts the necessary values from `formatOptions` for the proper response. In this case, we want to present the `enum` elements as a top-level `elements` key, and not simply returning them inside `formatOptions`